data/biobank-directory/molgenis.csv

```diff
@@ -1,202 +1,314 @@
+Collections,,collaboration_commercial,Commercial Collaboration,bool,,false,,The collection can be used for collaboration with commercial partners.,,,,,,,,,
+Collections,,collaboration_non_for_profit,Non-for-profit Collaboration,bool,,false,,The collection can be used for collaboration with non-for-profit partners.,,,,,,,,,
+Collections,,data_use,Data Use Ontologies,ontology_array,,false,,Data Use Ontologies (DUO),DirectoryOntologies,DataUseOntologies,,${label},,,,,
+Collections,,duc_profile,DUC profile,hyperlink,,false,,Link to the Digital Use Conditions (DUC) profile,,,,,,,,,
+Collections,,commercial_use,Available for Commercial Use,bool,,false,,Can commercial users request access to samples and/or data?,,,,,,,,,
+Collections,,access_fee,Fee-based access to,ontology_array,,false,,"Denotes whether access to samples, data or images may be obtained on fee-based basis.",DirectoryOntologies,AccessTypes,,${label},,,,,
+Collections,,access_joint_project,Access via Join Projects to,ontology_array,,false,,"Denotes whether access to samples, data or images may be obtained on joint project basis.",DirectoryOntologies,AccessTypes,,${label},,,,,
+Collections,,access_description,Access Description,text,,false,,Short description of the access rules.,,,,,,,,,
+Collections,,access_uri,Access URI,hyperlink,,false,,URI describing the access policy.,,,,,,,,,dcterms:accessRights
+Collections,,sop,PD/SOPs,ontology_array,,false,,Availability of Process Descriptions (PDs) and/or Standard Operating Procedures (SOPs).,DirectoryOntologies,SOPs,,${label},,,,,
+Collections,,rdf_helpers,RDF helpers,heading,,false,,Helper attributes to make RDF output compliant with various standards.,,,,,,,,,
+Collections,,catalog,,ref_array,,,,The FDP catalog(s) to which this collection belongs.,,Catalogs,,,,,,,ldp:membershipResource
+Collections,,license,,hyperlink,,false,,A legal document under which this catalogue record is made available.,,,,,,,,,dcterms:license
+Collections,,publisher,,ref,,false,,An entity responsible for making the resource available.,,Publishers,,,,,,,dcterms:publisher
+Collections,,language,,hyperlink_array,,false,,The language(s) of the resource.,,,,,,,,,dcterms:language
+Collections,,keywords,,string_array,,false,,Keyword(s) describing the resource.,,,,,,,,,dcat:keyword
+Collections,,EJP_RD_personalData,,string,,,,"Set to true if the resource onboarded to the EJP-RD Virtual Platform contains personal data, personal data meaning data related to identified or identifiable persons (as per GDPR definition), otherwise false. Must be String as workaround for current FDP reference implementation limitations.",,,,,,,,,ejp:personalData
+Collections,,EJP_RD_vpConnection,,hyperlink,,,,This property must be attached to every entity that you wish the EJP-RD Virtual Platform to explore.,,,,,,,,,ejp:vpConnection
+Collections,,issued,,datetime,,true,,When the metadata for this resource was first published,,,,,,,,mg_insertedOn,"dcterms:issued,fdp-o:metadataIssued"
```

Comment:
In DCAT dct:issued and dct:modified refer to the publication of the dataset, not of its metadata.